job/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -449,6 +450,11 @@ private void finish(@Nonnull Result r, @CheckForNull Throwable t) {
             }
         }
         FlowExecutionList.get().unregister(new Owner(this));
+        try {
+            StashManager.maybeClearAll(this);
+        } catch (IOException x) {
+            LOGGER.log(Level.WARNING, null, x);
```

Comment:
yes but these internal errors should always say what the error is about so the poor admin knows that he can probably ignore this as it's not the cause of the issue he is looking for.